src/System.Runtime/tests/Performance/Perf.String.cs

```diff
@@ -277,5 +277,620 @@ public void Trim_NothingToDo(int size)
+                using (iteration.StartMeasurement())
+                    s.Remove(i);
+        }
+
+        [Benchmark]
+        [InlineData("a", 0, 0)]
+        [InlineData("  ", 0, 1)]
+        [InlineData("  ", 1, 0)]
+        [InlineData("TeSt!", 0, 2)]
+        [InlineData("TeSt!", 2, 1)]
+        [InlineData("TeSt!", 3, 0)]
+        [InlineData("I think Turkish i \u0131s TROUBL\u0130NG", 0, 3)]
+        [InlineData("I think Turkish i \u0131s TROUBL\u0130NG", 18, 3)]
+        [InlineData("I think Turkish i \u0131s TROUBL\u0130NG", 22, 1)]
+        [InlineData("dzsdzsDDZSDZSDZSddsz", 0, 8)]
+        [InlineData("dzsdzsDDZSDZSDZSddsz", 7, 4)]
+        [InlineData("dzsdzsDDZSDZSDZSddsz", 10, 1)]
+        [InlineData("a\u0300\u00C0A\u0300A", 0, 2)]
+        [InlineData("a\u0300\u00C0A\u0300A", 3, 1)]
+        [InlineData("a\u0300\u00C0A\u0300A", 4, 0)]
+        [InlineData("Foo\u0400Bar!", 0, 4)]
+        [InlineData("Foo\u0400Bar!", 3, 2)]
+        [InlineData("Foo\u0400Bar!", 4, 1)]
+        [InlineData("a\u0020a\u00A0A\u2000a\u2001a\u2002A\u2003a\u2004a\u2005a", 0, 2)]
+        [InlineData("a\u0020a\u00A0A\u2000a\u2001a\u2002A\u2003a\u2004a\u2005a", 3, 3)]
+        [InlineData("\u4e33\u4e65 Testing... \u4EE8", 0, 3)]
+        [InlineData("\u4e33\u4e65 Testing... \u4EE8", 4, 1)]
+        public static void Remove_IntInt(string s, int i1, int i2)
+        {
+            foreach (var iteration in Benchmark.Iterations)
+                using (iteration.StartMeasurement())
+                    s.Remove(i1, i2);
+        }
+
+        [Benchmark]
+        [MemberData(nameof(ReplaceCharArgs))]
+        public static void Replace_Char(string s, char[] chars)
+        {
+            char c1 = chars[0];
+            char c2 = chars[1];
+
+            foreach (var iteration in Benchmark.Iterations)
+                using (iteration.StartMeasurement())
+                    s.Replace(c1, c2);
+        }
+
+        [Benchmark]
+        [MemberData(nameof(ReplaceStringArgs))]
+        public static void Replace_String(string s0, string[] strings)
+        {
+            string s1 = strings[0];
+            string s2 = strings[1];
+
+            foreach (var iteration in Benchmark.Iterations)
+                using (iteration.StartMeasurement())
+                    s0.Replace(s1, s2);
+        }
+
+        [Benchmark]
+        [InlineData("The quick brown fox", "The quick brown fox")]
+        [InlineData("Th\u00e9 quick brown f\u00f2x", "Th\u00e9 quick brown f\u00f2x")]
+        [InlineData("Th\u00e9 quick brown f\u00f2x", "Th\u00e9 quick brown f\u00f2x jumped")]
+        [InlineData("a\u0300a\u0300a\u0300", "\u00e0\u00e0\u00e0")]
+        [InlineData("a\u0300a\u0300a\u0300", "\u00c0\u00c0\u00c0")]
+        public static void Compare_CurrentCulture(string s1, string s2)
+        {
+            foreach (var iteration in Benchmark.Iterations)
+                using (iteration.StartMeasurement())
+                    String.Compare(s1, s2);
```

Comment:
This one makes me pause, because it's possible that a developer who has a different default culture will see different results than I'd see.  However, I like the idea of capturing current culture differences in build-to-build runs on stable hardware.  So, I'm inclined to keep this as is.

Of course, the origin here is that we ran this in a stable lab whose culture was always en-us.

Do you have concerns leaving this as is?